apis/duck/const.go

```diff
@@ -21,5 +21,11 @@ const (
 	// resources to exclude them from consideration when binding things.
 	// It is critical that bindings dealing with Deployments label their
 	// controller Deployment (or enclosing namespace).
+	// This is used for 'opt-in' behaviour (the default).
```

Comment:
The comment is kind of confusing label is `exclude`, but the behaviour is `opt-in`. A different phrasing can help, I think.

Reply:
Thanks! Tried to clarify, @vagababov PTAL.